genx/setup.cfg

```diff
@@ -1,14 +1,51 @@
-[sdist]
-formats=gztar
+[metadata]
+name = GenX
+version = attr: genx.version.__version__
+author = Matts Bjorck, Artur Glavic
+author_email = artur.glavic@gmail.com
+description = X-ray and Neutron reflectivity fitting software.
+long_description = file: README.txt
+url = http://genx.sourceforge.net/
+license = GPL v3
+license_file = LICENSE.txt
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: GNU General Public License v3 (GPLv3)
+    Operating System :: OS Independent
+    Topic :: Scientific/Engineering :: Physics
+    Development Status :: 6 - Mature
 
-[bdist]
-formats=rpm
 
-[bdist_rpm]
-requires=python3
-packager = Artur Glavic <artur.glavic@psi.ch>
+[options]
+python_requires = >=3.6
+packages =
+    genx
+    genx.lib
+    genx.models
+    genx.models.lib
+    genx.plugins
+    genx.plugins.add_ons
+    genx.plugins.add_ons.help_modules
+    genx.plugins.data_loaders
+    genx.plugins.data_loaders.help_modules
+scripts = scripts/genx
+install_requires =
+    numpy
+    matplotlib
+    scipy
+    appdirs
+    wxPython
+    h5py
 
-[bdist_wininst]
 
-[install]
-optimize=1
+[options.package_data]
+genx =
+    genx.conf
+    examples/*.*
+    LICENSE.txt
+    changelog.txt
```

Comment:
changelog.txt and LICENSE.txt should be in the root folder, not the genx package. (genx here is relative to setup.py, thus it corresponds to genx/genx/ in the repository.)
As you have created a license file already in the base folder I think it makes sense to put all of that in the base folder to separate from the package or all of it in the package.